steps/pagerduty/0.1.0/step.yml

```diff
@@ -0,0 +1,110 @@
+title: PagerDuty
+summary: |
+  Create a PagerDuty alert
+description: |
+  This step creates a PagerDuty alert with specific parameters. It can use Events API v1 or v2.
+website: https://github.com/PagerDuty/bitrise-step-pagerduty
+source_code_url: https://github.com/PagerDuty/bitrise-step-pagerduty
+support_url: https://github.com/PagerDuty/bitrise-step-pagerduty/issues
+published_at: 2019-09-16T16:15:01.059024-04:00
+source:
+  git: https://github.com/PagerDuty/bitrise-step-pagerduty.git
+  commit: 1b1bf9ab212df4dd745f581d64d6941cf9d84a5b
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  - name: jq
+  apt_get:
+  - name: git
+  - name: wget
+  - name: curl
```

Comment:
Curl should be required always, not only on linux since it looks like to be used regardless of OS.

Reply:
`curl` is pre-installed on macOS. If you do `$ brew install curl` you will get this message:

```
curl is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.
```

Should I add it anyway?

Reply:
Other steps added it so I'll add it too, even if it's not the curl being used.